qiskit/visualization/matplotlib.py

```diff
@@ -759,8 +759,12 @@ def _draw_ops(self, verbose=False):
                     self._condition(node, cond_xy)
 
                 # draw measure
-                if isinstance(op, Measure):
-                    self._measure(node)
+                if isinstance(op, (Measure, MeasurePauli)):
+                    if isinstance(op, Measure):
```

Comment:
This is an example of a bug mentioned in https://github.com/Qiskit/qiskit-terra/pull/5311/files#r810538319
```suggestion
                    if isinstance(op, Measure):
```